gen_gold_lf.py

```diff
@@ -52,7 +52,18 @@ def process_cat(fpath, vmax_opath, field=None):
     ##  Luminosity fn.
     opath = opath.replace('vmax', 'lumfn')
 
-    VV      = volcom(gama_vmax['ZGAMA'].max(), Area) - volcom(gama_vmax['ZGAMA'].min(), Area)
+    
+    if density_split == True:
+        
+        rand_path = '/global/cscratch1/sd/ldrm11/desi/BGS/Sam/randoms_bd_ddp_n8_{}_0.fits'.format(field)
+        rand = Table.read(rand_path)
+        scale = rand.meta['DDP1_d{}_VOLFRAC'.format(idx)]
+        VV = vol_normalise(gama_vmax, scale)
+        
+    else:
+        VV = volcom(gama_vmax['ZGAMA'].max(), Area) - volcom(gama_vmax['ZGAMA'].min(), Area)
+
+            
     result  = lumfn(gama_vmax, VV)
```

Comment:
whoa, this is way off!  Leave VV alone, it's the actual surveyed volume.  

It's 
result = lumfn(gama_vmax, VV)
result = vol_normalise(result, scale)

where we've renamed vol_normalise as for the previous comment.